index.html

```diff
@@ -37,6 +39,147 @@ <h2 class="connect">LET’S CONNECT</h2>
 
 </div>
 
+<section class="works">
+  <article class="work1">
+    <img src="work1-img.svg" alt="Work image">
+    <div class="work-text-container">
+      <header class="work-header">Tonic</header>
+      <ul class="work-subheader">
+        <li class="list1">CANOPY</li>
+        <li class="dot"></li>
+        <li class="list2">Back End Dev</li>
+        <li class="dot"></li>
+        <li class="list2">2015</li>
+      </ul>
+      <p class="work-text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      <ul class="program-list">
+        <li>html</li>
+        <li>css</li>
+        <li>javascript</li>
+      </ul>
+      <nav><input type="button" value="See Project" /></nav>
+    </div>
+  </article>
+
+  <article class="work2">
```

Comment:
- [ ] Kindly fix the height of the card, the button see project is too close

![Capturesee](https://user-images.githubusercontent.com/81444338/176997043-8937254b-e303-4367-bd70-0393694bc401.PNG)